layouts/index.html

```diff
@@ -25,6 +25,14 @@ <h3>{{ .Site.Params.Description }}</h3>
 </section>
 {{ end }}
 
+{{ if isset .Site.Params "enablefrontmatter" }}
+    {{ if eq .Site.Params.EnableFrontMatter "true" }}
+    <section class="home-content">
```

Comment:
And an `<hr>` tag before section

Reply:
1bcafd01e127cfb2dafcce2813eb73bd5b1a8caa